modules/client/src/interfaces.ts

```diff
@@ -553,20 +555,57 @@ export type SubgraphPluginRepoListItem = {
   releases: SubgraphPluginRepoReleaseListItem[];
 };
 
+export type SubgraphPluginVersion = {
+  build: number;
+  metadata: string;
+  release: {
+    release: number;
+    metadata: string;
+  };
+  pluginRepo: {
+    id: string;
+    subdomain: string;
+  };
+};
+
 export type PluginRepoReleaseMetadata = {
   name: string;
   description: string;
   images: Object; // TODO specify parameters
 };
+export type PluginRepoBuildMetadata = {
+  ui: string;
+  change: string;
+  pluginSetupABI: Object;
+};
 
 export type PluginRepoRelease = {
   release: number;
   metadata: PluginRepoReleaseMetadata;
-  builds: number[];
+  latestBuild: number;
```

Comment:
Soft suggestion, no strong feelings

```suggestion
  currentBuild: number;
```